app/services/authorization/enterprise_service.rb

```diff
@@ -37,7 +37,7 @@ def initialize(token)
   # Return a true ServiceResult if the token contains this particular action.
   def call(action)
     allowed =
-      if token.nil? || token.token_object.nil? || token.expired?
+      if OpenProject::Configuration.ee_manager_visible? && (token.nil? || token.token_object.nil? || token.expired?)
```

Comment:
This shouldn't influence the service result. It should still return the correct check for a specific feature.